pkg/cmd/validation/uuid.go

```diff
@@ -0,0 +1,12 @@
+package validation
+
+import "github.com/google/uuid"
+
+// UUID validates that the given string is a valid UUID.
+func ValidateUUID(s string) error {
```

Comment:
I'm not sure the value in this, either we'll get an error from the API or can validate it early.

But I think it is useful for users to be able to use the agent URL as well and we parse that out. So they can easily copy/paste from the web

Reply:
Hmm, my personal idea with the CLI is that it reduces/entirely removes the requirement to go to the web. I don't really see much value in a CLI in which you need to go to the web to get the values.